### PR TITLE
Update OTEDetection

### PR DESCRIPTION
**Pull request by druzhkov-paul**
Update OTEDetection to enable OpenVINO 2021.3 support for detection, instance segmentation and text spotting tasks.

### CODE REVIEW

**Comment by druzhkov-paul** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
run tests

**Comment by druzhkov-paul** [comment]
@LeonidBeynenson @aleksei-kashapov Please note, that this PR disables FaceDetection0207NNCFint8 test as it is not fully-compatible with OpenVINO 2021.3, which is used for CI runs by default now.

File: models/object_detection/tests/nncf_tests_face_detection.py [hunk 1]
```diff
@@ -90,16 +90,18 @@ class FaceDetection0204NNCFint8(
...
-            'face-detection',
-            'face-detection-0207',
-            os.path.dirname(__file__) + '/../../../data/airport/annotation_faces_train.json',
-            os.path.dirname(__file__) + '/../../../data/airport/',
-            '--nncf-quantization'
-        )
-):
-    """ Test case for face-detection-0207 model with NNCF int8 compression. """
+
+# # Quantized model is crashing at OpenVINO inference. Disabling the test until the bugfix.
```
Comment (Ilya-Krylov) [inline models/object_detection/tests/nncf_tests_face_detection.py:94]: FYI @LeonidBeynenson @aleksei-kashapov

**Review by Ilya-Krylov — APPROVED** [review]

**Comment by aleksei-kashapov** [comment]
> @LeonidBeynenson @aleksei-kashapov Please note, that this PR disables FaceDetection0207NNCFint8 test as it is not fully-compatible with OpenVINO 2021.3, which is used for CI runs by default now.

Thank you for mention this! Also, @LeonidBeynenson can we extend models scope in our CI validation process as we added int8 quantization support for many new models

**Pull request merged**